framework/src/multiapps/MultiApp.C

```diff
@@ -521,6 +522,9 @@ MultiApp::preTransfer(Real /*dt*/, Real target_time)
     _reset_happened = true;
     for (auto & app : _reset_apps)
       resetApp(app);
+
+    for (auto * const transfer : _associated_transfers)
```

Comment:
because I removed `getAppInfo` from all the `execute` methods and moved it to `initialSetup`. That change if fine unless an app gets reset in which case you get to restore the app info